grammars/README.md

```diff
@@ -9,6 +9,8 @@ an index.
 
 - [Ace](https://ace.c9.io/) — supported. The grammar is upstream
   ([prql_highlight_rules.js](https://github.com/ajaxorg/ace/blob/master/src/mode/prql_highlight_rules.js)).
+  See the
+  [demo](https://htmlpreview.github.io/?https://github.com/PRQL/prql/blob/main/grammars/ace-demo.html).
```

Comment:
Does this work? I'm not sure I'm doing it right but if I try https://htmlpreview.github.io/?https://github.com/PRQL/prql/blob/0bcb3d530aa9eb411a9c09380738732f5be1407f//grammars/ace-demo.html, I just get some text.

Reply:
Ugh, I had hoped so, but I guess not. I can move to `/web/website/content/` though.

Reply:
OK, let's try that

Reply:
(need to update link too :) )

Reply:
I hope this works!